app/src/main/res/raw/data.json

```diff
@@ -1738,5 +1738,2365 @@
+    ],    
+	[
+    {"EN":{
+        "word": "sea",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "море",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "ocean",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "океан",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "hotel",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "отель",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "borders",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "границы",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "customs",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "таможня",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "station",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "станция",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "airport",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "аэрпорт",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "beach",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "пляж",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "museum",
+        "topics": ["travel"]
+        }},
+    {"RU": {
+        "word": "музей",
+        "topics": ["путешествия"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "sweater",
+        "topics": ["dress"]
+        }},
+    {"RU": {
+        "word": "свитер",
+        "topics": ["одежда"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "jacket",
+        "topics": ["dress"]
+        }},
+    {"RU": {
+        "word": "куртка",
+        "topics": ["одежда"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "coat",
+        "topics": ["dress"]
+        }},
+    {"RU": {
+        "word": "пальто",
+        "topics": ["одежда"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "jeans",
+        "topics": ["dress"]
+        }},
+    {"RU": {
+        "word": "джинсы",
+        "topics": ["одежда"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "socks",
+        "topics": ["dress"]
+        }},
+    {"RU": {
+        "word": "носки",
+        "topics": ["одежда"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "shorts",
+        "topics": ["dress"]
+        }},
+    {"RU": {
+        "word": "шорты",
+        "topics": ["одежда"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "vest",
+        "topics": ["dress"]
+        }},
+    {"RU": {
+        "word": "жилет",
+        "topics": ["одежда"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "pajamas",
```

Comment:
Do you use American English? `pyjamas` is also correct and in ideal world it shouldn't be a mistake, but it will require additional logic

Reply:
I think american, but it is not so critical for me.